nistats/reporting/glm_reporter_templates/report_template.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta name="viewport" content="width=device-width, inital-scale=1, shrink-to-fit=yes">
+    <meta charset="utf-8">
+    <title>${page_title}</title> <!-- func:glm_reporter._make_headings() -->
+
+    <style>
```

Comment:
the reports look good, congrats @kchawla-pi !

Reply:
Thanks! :-)